pkg/virtual/workspaces/builder/build.go

```diff
@@ -145,7 +145,7 @@ func BuildVirtualWorkspace(cfg *clientrest.Config, rootPathPrefix string, wildca
 						return nil, err
 					}
 
-					workspacesRest := registry.NewREST(cfg, kcpClusterClient.Cluster(tenancyv1alpha1.RootCluster).TenancyV1alpha1(), kubeClusterClient, kcpClusterClient, globalClusterWorkspaceCache, crbInformer, orgListener.FilteredClusterWorkspaces)
+					workspacesRest := registry.NewREST(cfg, kubeClusterClient, kcpClusterClient, globalClusterWorkspaceCache, crbInformer, orgListener.FilteredClusterWorkspaces)
```

Comment:
where is the counterpart of the RootCluster here? Where did that go?

Reply:
`RootClusterClient` wasn't being used at all (https://github.com/kcp-dev/kcp/blob/v0.6.3/pkg/virtual/workspaces/registry/rest.go) and was redundant. So removed it.

Reply:
ack. Left over from early implementation.